src/SpeedInput/speedInputStyles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from '@emotion/styled';
+import theme from 'src/styles/theme';
+
+export const RadioGroupWrapper = styled.div({
```

Comment:
We need to pass theme here like in other style components. styled.div(({theme})=>